packages/now-static-build/src/index.ts

```diff
@@ -1,5 +1,6 @@
 import ms from 'ms';
 import path from 'path';
+import fetch from 'node-fetch';
```

Comment:
Should we use `@zeit/fetch`?

Reply:
None of the builders use `@zeit/fetch`, they all seem to use `node-fetch`.

If we want to change it, we should probably change every package in the monorepo.